### ISSUE TITLE
[BUG] install failed to add user to group Hyper-V administrator

### ISSUE THREAD

**Issue by oluies** [code-ready/crc#2620]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V 
  * Did you run `crc setup` before starting it (Yes/No)? no 
  * Running CRC on: Laptop

## CRC version
```bash
# Put `crc version` output here

```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `systeminfo` in case of Windows

Host Name:                 NN
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.19042 N/A Build 19042
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free
Product ID:                00329-00000-00003-AA985
Original Install Date:     2021-02-24, 14:20:16
System Boot Time:          2021-08-02, 17:27:38
System Manufacturer:       HP
System Model:              HP ZBook Firefly 15 G7 Mobile Workstation
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 142 Stepping 12 GenuineIntel ~1803 Mhz
BIOS Version:              HP S73 Ver. 01.03.02, 2020-11-04
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume2
System Locale:             sv;Svenska
Input Locale:              sv;Svenska
Time Zone:                 (UTC+01:00) Amsterdam, Berlin, Bern, Rom, Stockholm, Wien
Total Physical Memory:     65 309 MB
Available Physical Memory: 59 401 MB
Virtual Memory: Max Size:  81 693 MB
Virtual Memory: Available: 75 798 MB
Virtual Memory: In Use:    5 895 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    NNN.SE
Hotfix(s):                 10 Hotfix(s) Installed.
                           [01]: KB5003537
                           [02]: KB4562830
                           [03]: KB4570334
                           [04]: KB4577586
                           [05]: KB4580325
                           [06]: KB4589212
                           [07]: KB4593175
                           [08]: KB4598481
                           [09]: KB5004237
                           [10]: KB5003742
```

### Steps to reproduce

  1.  Start install 
  2. 
![image](https://user-images.githubusercontent.com/150961/127888169-ac484d7c-6288-4002-9016-c0e8a5432b07.png)

  4. My user and admin user is in the Hyper-V administrator already. The user is in a domain 
```bash
PS C:\Users\TT> Get-LocalGroupMember -Name crc-users -Member 'NNN\TT'

ObjectClass Name     PrincipalSource
----------- ----     ---------------
User        NNN\TT ActiveDirectory

PS C:\Users\TT> Get-LocalGroupMember -Name crc-users -Member 'TT'
Get-LocalGroupMember : Principal TTwas not found.

Get-LocalGroupMember -Group "Hyper-V Administrators"
Get-LocalGroupMember : Group Hyper-V Administrators was not found.

Get-LocalGroupMember -Group "Hyper-V-Administratörer"
ObjectClass Name        PrincipalSource
----------- ----        ---------------
User        nnn\tt    ActiveDirectory
User        nnn\ttadm ActiveDirectory

Get-LocalGroupMember -SID S-1-5-32-578

ObjectClass Name        PrincipalSource
----------- ----        ---------------
User        nnn\tt    ActiveDirectory
User        nnn\ttadm ActiveDirectory


```

### Expected

Ref #421

**Comment by anjannath** [comment]
@oluies Can you try the workaround mentioned in https://github.com/code-ready/crc/issues/2581#issuecomment-889772328

Duplicate of https://github.com/code-ready/crc/issues/2581

**Comment by oluies** [comment]
LGTM, Duplicate of #2581, works for me.  Lokks like it is fixed? https://github.com/code-ready/crc/blob/8928cf1e76b5e74be1506cd664c7a0be7955871d/pkg/crc/preflight/preflight_checks_windows.go#L134

**Comment by mraufer** [comment]
I have the same problem (localized Windows where group names are different) - the workaround #2581 did work but is is and should be just a workaround:
- whatever the reasons for localized group names, renaming them might cause issues elsewhere that are not as easy to trouble-shoot; do we need to rename it back to the localized version so that everything else will work? will crc then fail again?
- why does crc need to mess with Hyper-V at all? at least let users do those changes manually that cause trouble ...
- I had a previous version of crc on my system and thus my user was already a member of the group, crc just could not figure that out due to localization
- a good installer is able to handle localization
Please update your installer to handle localization, or to not require admin actions on Hyper-V, or to tell users what to do manually to prepare the system for the installer to work properly.
Thanks.

**Comment by anjannath** [comment]
@mraufer @oluies This issue is not fixed yet, It is not possible to have localized group names in the msi source, and we want to create an experience for the user where he has to do minimum or no manual setup. we are trying to fix this issue and will be tracked in #2581

**Comment by gbraad** [comment]
Duplicate of #2581

**Comment by oluies** [comment]
> @mraufer @oluies This issue is not fixed yet, It is not possible to have localized group names in the msi source, and we want to create an experience for the user where he has to do minimum or no manual setup. we are trying to fix this issue and will be tracked in #2581

One should use the well known SID for the group instead of the name eg BUILTIN\Hyper-V Administrators => S-1-5-32-578
https://support.microsoft.com/en-us/help/243330/well-known-security-identifiers-in-windows-operating-systems

**Comment by oluies** [comment]
I have no idea what is used, but mayby Wixtoolkit 

https://github.com/code-ready/crc/blob/5ea8c1ca0bf24e0d5b5bd459dda48a9aec00f1bc/packaging/windows/product.wxs.in#L59

Looks like there is no way to reference the well know sid in Wix?
